types/applepayjs/index.d.ts

```diff
@@ -39,6 +39,11 @@ declare class ApplePaySession extends EventTarget {
      */
     onvalidatemerchant: (event: ApplePayJS.ApplePayValidateMerchantEvent) => void;
 
+    /**
+     * A callback function that is automatically called when the coupon code is changed.
```

Comment:
```suggestion
     * An event handler called by the system when the user enters or updates a coupon code.
```

Reply:
Done, thanks!
Could you please take a look?